packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -815,6 +815,16 @@ abstract class FlutterCommand extends Command<void> {
     );
   }
 
+  void addMultiDexOption({ bool hide = false }) {
+    argParser.addFlag('multi-dex',
```

Comment:
as weird as it sounds, most docs I found refer to "multidex" as a single word.

Reply:
I was debating to use the one word vs two word version. I think technically, it is two, but I did notice that a lot of docs used the single word, so I will probably just switch to the single word to remain consistent.